pkgs/applications/networking/cluster/argocd-autopilot/default.nix

```diff
@@ -0,0 +1,53 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "argocd-autopilot";
+  version = "0.2.25";
+  commit = "b769e2e9886c1355aedffee42077245d120ff4f1";
```

Comment:
is the commit really necessary?

For the nixpkgs-update machinery, it most likely wont update this, and it will get out-of-date

Reply:
Commit hash is built into the binary and is reported by the version command. I can replace it with "unknown", similarly to build date if that's better.

Reply:
@jonringer what do you think would be the better option.

Reply:
+1 for removing it and leaving just version, I was using this the same way and was way too boring to update the PR from bot.